client/src/pages/ErrorPage/ErrorPage.module.scss

```
@@ -0,0 +1,44 @@
+.container {
+    position: relative;
+    width: 100%;
+    height: 100%;
+    text-align: center;
+    & > img {
```

Don't you think that there is no reason to use child selector here and in other places?
```suggestion
    img {
```